cloudinit/net/netplan.py

```diff
@@ -430,13 +430,17 @@ def _render_content(self, network_state: NetworkState) -> str:
             # filter None (but not False) entries up front
             ifcfg = dict(filter(lambda it: it[1] is not None, config.items()))
 
+            logging.debug("config for %s:\n%s", ifname, ifcfg)
```

Comment:
This is the network state log, correct? I don't think this will be helpful in the cloud-init logs.